hybrasyl/NumberCruncher.cs

```diff
@@ -145,7 +145,7 @@ public static double CalculateHeal(Castable castable, ModifierEffect effect, Cre
 
             if (effect?.Heal == null) return heal;
 
-            var statusAdd = castable?.Statuses?.Add?.Where(e => e.Value == statusName)?.ToList();
+            var statusAdd = castable?.Effects?.Statuses?.Add?.Where(e => e.Value == statusName)?.ToList();
             var intensity = statusAdd != null ? statusAdd[0].Intensity : 1;
```

Comment:
this appears to only check the first intensity of a list of statuses. This must be corrected.

Reply:
You cannot add the same status multiple times. Damage from each tick would be checked against its own intensity, since it is possible to add multiple statuses with different intensities.

Reply:
Right, but you can add multiple separate statuses from a castable. So this would only check the first intensity, and ignore the rest of the Add statuses.

Reply:
Please note I am finding the intensity for the current effect (`e.Value == statusName`). I do not need to care about other intensities here; they are handled on their own. `CalculateHeal / Damage` is run individually for each tick of a status. For instance, if I had a heal over time applied, I would rely on its defined intensity in the castable. I don't need to know the intensity of anything else (say, a poison).

There should never be an instance of adding the same named status twice with differing intensities; that makes no sense (and also isn't possible with current client limitations)